examples/tornjak/keycloak/README.md

```diff
@@ -4,111 +4,66 @@ This example demonstrates Tornjak's capability to control access to the Frontend
 User Management via [Keycloak](https://www.keycloak.org/).
 
 Tested on:
+
 - Keycloak Application Version - 24.0.3
 - Keycloak Chart Version - 21.0.3
 
 For more information regarding Tornjak User Management, please refer to the following documentation:
 
-* [Tornjak User Management](https://github.com/spiffe/tornjak/blob/main/docs/keycloak-configuration.md)
-* [Keycloak Configuration for Tornjak](https://github.com/spiffe/tornjak/blob/main/docs/keycloak-configuration.md)
-* [Detailed Blogs on Tornjak User Management](https://github.com/spiffe/tornjak/blob/main/docs/blogs.md)
+- [Tornjak User Management](https://github.com/spiffe/tornjak/blob/main/docs/keycloak-configuration.md)
+- [Keycloak Configuration for Tornjak](https://github.com/spiffe/tornjak/blob/main/docs/keycloak-configuration.md)
+- [Detailed Blogs on Tornjak User Management](https://github.com/spiffe/tornjak/blob/main/docs/blogs.md)
 
-**NOTE:** This example works only with the Vanilla version of Kubernetes; it does not yet support Openshift.
+> [!NOTE]
+> This example works only with the Vanilla version of Kubernetes; it does not yet support Openshift.
 
 As part of the exercise, an instance of Keycloak is deployed to illustrate how to manage users' access to Tornjak.
 Once enabled, the Tornjak UI will redirect all authentication calls to the Keycloak instance to obtain the
 correct credentials. Authorization is based on these credentials and occurs at the Tornjak application level.
 
 ## Deploy Keycloak Instance (Authentication Service)
 
-We will deploy the instance of Keycloak in the same namespace as the SPIRE Server
+We will deploy the instance of Keycloak in the same namespace as the SPIRE Server,
```

Comment:
Update:
We will create a namespace and deploy the instance of Keycloak in the that namespace.